src/form/bf_mx.jl

```diff
@@ -345,8 +150,8 @@ end
 function constraint_tp_theta_ref(pm::_PMs.GenericPowerModel{T}, n::Int, i) where T <: AbstractUBFForm
     nconductors = length(_PMs.conductor_ids(pm))
 
-    w_re = _PMs.var(pm, n, :W_re)[i]
-    w_im = _PMs.var(pm, n, :W_im)[i]
+    w_re = _PMs.var(pm, n, :Wr)[i]
```

Comment:
I think variable names should be consistent with the JuMP variable names, e.g. `Wr`